pallets/ddc-clusters/src/cluster.rs

```diff
@@ -48,6 +50,7 @@ impl<AccountId> Cluster<AccountId> {
 				},
 				node_provider_auth_contract: cluster_params.node_provider_auth_contract,
 			},
+			extension_smart_contract,
```

Comment:
Let's use the `node_provider_auth_contract` field from the `cluster_params`. This struct contains all parameters that can be set by a Cluster Manager without the Governance approval. 
Changes for `node_provider_auth_contract` are not needed to be approved by the Governance, so it can be added to `node_provider_auth_contract`

Reply:
Thank you, updated